introduction/Infrastructure.tex

```diff
@@ -9,23 +9,42 @@ \section{Infrastructure}
 \subsection{RoboCup@Home Mailinglist}
 The official \iterm{RoboCup@Home mailing list} can be reached at
 \begin{center}
-\texttt{robocup-athome@lists.robocup.org}
+\href{mailto:robocup-athome@lists.robocup.org}{\texttt{robocup-athome@lists.robocup.org}}
 \end{center}
 You can register to the email list at:
 \begin{center}
-http://lists.robocup.org/cgi-bin/mailman/listinfo/robocup-athome
+{\small\url{http://lists.robocup.org/cgi-bin/mailman/listinfo/robocup-athome}}
 \end{center}
 
 \subsection{RoboCup@Home Web Page}
-The official \iterm{RoboCup@Home website} that also hosts this RuleBook can be found at \\
+The official \iterm{RoboCup@Home website} that also hosts this RuleBook can be found at:
 \begin{center}
-http://www.robocupathome.org/
+{\small\url{https://athome.robocup.org/}}
+\end{center}
+
+\subsection{RoboCup@Home Rulebook Repository}
+The official \iterm{RoboCup@Home Rulebook Repository} is where rules are publicly discussed before applying changes to the rulebook.
+The entire RoboCup@Home community is welcome and encouraged to actively participate in creating and discussing the rules. The repository can be reached at:
+\begin{center}
+{\small\url{https://github.com/RoboCupAtHome/RuleBook/}}
+\end{center}
+
+Although opening issues with inconsistencies, questions, clarifications, and suggestions is highly appreciated, the best way to contribute is by making pull requests with fixes and proposed changes.
+
+\subsection{RoboCup@Home Telegram Group}
+The official \iterm{RoboCup@Home Telegram Group} is and communication channel for the RoboCup@Home community where rules are discussed, announcements are made, and questions are answered.
+However, beyond the technical aspects of the competition, the \textit{Telegram Group} is a meeting point to stay in contact with the community, foster knowledge exchange and strengthen relationships.
```

Comment:
Maybe: In addition to discussing technical aspects of the competition, ...

However sounds like this is opposed to what was stated before.

Reply:
Any suggestions?

Reply:
```suggestion
Beyond the technical aspects of the competition, the \textit{Telegram Group} is a meeting point to stay in contact with the community, foster knowledge exchange and strengthen relationships.
```